MapImageEngine/src/FaigerSYS/MapImageEngine/storage/MapImageChunk.php

```diff
@@ -130,10 +130,10 @@ public function getABGR(int $x, int $y) : int {
 	 */
 	public function setABGR(int $x, int $y, int $color) {
 		$pos = $this->getStartOffset($x, $y);
-		$this->data->buffer{$pos++} = chr($color >> 24 & 0xff);
-		$this->data->buffer{$pos++} = chr($color >> 16 & 0xff);
-		$this->data->buffer{$pos++} = chr($color >> 8  & 0xff);
-		$this->data->buffer{$pos}   = chr($color       & 0xff);
+		$this->data->buffer[$pos++] = chr($color >> 24 & 0xff);
+		$this->data->buffer[$pos++] = chr($color >> 16 & 0xff);
+		$this->data->buffer[$pos++] = chr($color >> 8  & 0xff);
+		$this->data->buffer[$pos] = chr($color       & 0xff);
```

Comment:
Here you removed two spaces before the `=` sign that were present to make the last `chr` call aligned with the previous ones.

Reply:
i just used code styles from pmmp. does this affect the final work of the plugin?

Reply:
No but you shouldn't change the formatting of the code for no reason at all

Reply:
i changed the formatting back, the author hasn't been here for a long time, so I think nothing will change.